website/content/en/docs/reference/threat-model.md

```diff
@@ -29,92 +28,161 @@ A Cluster Developer has the ability to create pods via `Deployments`, `ReplicaSe
+
+#### Karpenter can be used to create or terminate EC2 instances outside the cluster
+
+**Threat:** EC2 instances can exist in an AWS account outside of the Kubernetes cluster. An actor who obtains control of the Karpenter pod’s IAM role may be able to create or terminate EC2 instances not part of the Kubernetes cluster managed by Karpenter.
+
+**Mitigation:** Karpenter creates instances with tags, several of which are enforced in the IAM policy granted to the Karpenter IAM role that restrict the instances Karpenter can terminate. One tag requires that the instance was provisioned by a Karpenter controller (`karpenter.sh/nodepool`), another tag can include a cluster name to mitigate any termination between two clusters with Karpenter in the same account `kubernetes.io/cluster/${CLUSTER_NAME}`. Cluster Operators also can restrict the region to prevent two clusters in the same account with the same name in different regions. The IAM permissions provided to customers restrict region out of the box by default. Additionally, Karpenter does not allow tags to be created or modified on instances unowned by Karpenter after creation, except for the `Name` and `karpenter.sh/nodeclaim` tags. Though these tags can be changed after instance creation, `aws:ResourceTag` conditions enforce that the Karpenter controller is only able to change these tags on instances that it already owns, enforced through the `karpenter.sh/nodepool` and `kubernetes.io/cluster/${CLUSTER_NAME}` tags.
+
+#### Karpenter can orchestrate the creation/deletion of IAM Instance Profiles it doesn’t own
+
+**Threat:** Karpenter has permission to create/update/delete instance profiles as part of its controller permissions to ensure that it can auto-generate instance profiles when `EC2NodeClasses` are created. An actor who obtains control of the Karpenter pod’s IAM role may delete instance profiles not owned by Karpenter, causing workload disruption to other instances using the profile in the account.
+
+**Mitigation:** Karpenter’s controller permissions enforce that it creates instance profiles with tags which provide ownership information. These tags include:
 
 * `karpenter.sh/managed-by`
 * `kubernetes.io/cluster/${CLUSTER_NAME}`
 * `karpenter.k8s.aws/ec2nodeclass`
 * `topology.kubernetes.io/region`
 
-These tags ensure that instance profiles created by Karpenter in the account are unique to that cluster. Karpenter's controller permissions _only_ allow it to act on instance profiles that contain these tags which match the cluster information.
+Source: https://karpenter.sh/docs/reference/cloudformation/#allowscopedinstanceprofilecreationactions
+
+These tags ensure that instance profiles created by Karpenter in the account are unique to that cluster. Karpenter’s controller permissions only allow it to act on instance profiles that contain these tags which match the cluster information.
```

Comment:
Can we use this same wording for instance permissions that we list above -- seems like similar wording in both of these locations would be nice